crates/bevy_reflect/derive/src/derive_data.rs

```diff
@@ -482,7 +482,7 @@ impl<'a> ReflectMeta<'a> {
         )
     }
 
-    /// The collection of docstrings for this type, if any.
```

Comment:
docstring is a word

Reply:
I'll add it to the spelling exceptions, then.